lib/jsonrpc-adapter.js

```diff
@@ -66,7 +66,7 @@ JsonRpcAdapter.prototype.getRoutes = function(obj) {
   return routes;
 };
 
-JsonRpcAdapter.errorHandler = function() {
+JsonRpcAdapter.strongErrorHandler = function() {
```

Comment:
What is the purpose of this API rename? Why can't we keep existing `errorHandler` name?